src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback/Application/Commands/GenerateEmailTransaction/GenerateEmailTransactionCommandHandler.cs

```diff
@@ -0,0 +1,30 @@
+﻿using MediatR;
+using SFA.DAS.ApprenticeFeedback.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.Extensions;
+using SFA.DAS.SharedOuterApi.Interfaces;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeFeedback.Application.Commands.GenerateEmailTransaction
+{
+    public class GenerateEmailTransactionCommandHandler : IRequestHandler<GenerateEmailTransactionCommand, GenerateEmailTransactionResponse>
+    {
+        private readonly IApprenticeFeedbackApiClient<ApprenticeFeedbackApiConfiguration> _feedbackApiClient;
+
+        public GenerateEmailTransactionCommandHandler(IApprenticeFeedbackApiClient<ApprenticeFeedbackApiConfiguration> feedbackApiClient)
```

Comment:
Any unit tests for the controller / handler ?

Reply:
Added unit tests